webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/PreferencesUtil.java

```diff
@@ -114,6 +125,14 @@ public static void loadPreferences(String aUsername, SettingsService aSettingsSe
                         .getProject());
                 aBModel.setAnnotationLayers(layers);
             }
+
+            // Get color preferences for each layer, init with legacy if not found
```

Comment:
last commit b20a5fd should account for that, at least it does during debugging. But it still doesn't show up in preferences because I'm iterating over `bModel.getAnnotationLayers()` and that doesn't seem to get the change. Might I suggest exchanging `List<Long> getAnnotationLayers()` in  `AnnotationPreference` with stg like `Map<Long, Boolean> getAnnotationLayerVisibility()`.